spec/unit/crypto/cross-signing.spec.ts

```diff
@@ -1023,7 +1068,7 @@ describe("Cross Signing", function() {
             },
         };
         alice.crypto.deviceList.storeDevicesForUser("@alice:example.com", {
-            Dynabook: aliceNotCrossSignedDevice,
+            Dynabook: aliceNotCrossSignedDevice as unknown as IDevice,
```

Comment:
Can we get a comment for why this is needed

Reply:
Fixed the object instead